packages/react-components/src/components/graph/graph.tsx

```diff
@@ -43,25 +43,31 @@ export const Graph: FC<CytoGraphProps> = ({ graphLayout, className, ...props })
     cy.current?.center();
   }, []);
 
+  // istanbul ignore next
+  // Don't care about testing reacts built in ref setting logic.
+  const setCy = (c: Core) => {
+    cy.current = c;
+  };
+
   return (
-    <Container className={`ak-graph ${className}`.trim()} {...props}>
+    <Container className={`ak-graph ${className || ''}`.trim()} {...props}>
       <CytoscapeComponent
-        cy={(c) => (cy.current = c)}
+        cy={setCy}
```

Comment:
Nit: This would require a larger change, but the name cy, which is colloquially a reference to Cypress, is a little confusing, especially in the context of testing (i.e., fakeCy).